.github/workflows/frontent-ci-pipeline.yml

```diff
@@ -0,0 +1,36 @@
+name: "Frontend test - action"
+
+on:
+  push:
+    paths:
+      - "frontend/**"
+  pull_request:
+    paths:
+      - "frontend/**"
+jobs:
+  test-frontend:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '16.13.2'
+
+      - name: Install dependencies 
+        run: |
+          cd frontend
+          npm ci
+      - name: Run lint
+        run: |
+          cd frontend
+          npm run lint
+      - name: Run build frontend
+        run: |
+          cd frontend
+          CI=false
```

Comment:
Why is it false? What effect does it have?

Reply:
This is to allowing a successful build when there are linting warnings, not necessary now.